guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -82,9 +82,16 @@ Reboot these hosts after the upgrade has completed.
 # {foreman-maintain} service stop
 ----
 
-. Obtain the latest ISO files by following the https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/installing-satellite-server-disconnected#downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in the _Installing {ProjectServer} from a Disconnected Network_ guide.
+. Obtain the latest ISO files by following the {InstallingDisconnectedDocURL}downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in _{InstallingServerDisconnectedDocTitle}_.
 
-. Create directories to serve as a mount point, mount the ISO images, and configure the `rhel7-server` repository by following the https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/installing-satellite-server-disconnected#configuring-the-base-operating-system-with-offline-repositories_satellite[Configuring the Base System with Offline Repositories] procedure in the _Installing {ProjectServer} from a Disconnected Network_ guide.
+. Create directories to serve as a mount point, mount the ISO images, and configure the `rhel7-server` or the `rhel8` repository:
++
+.For {EL} 8
+Follow the {InstallingDisconnectedDocURL}configuring-the-base-operating-system-with-offline-repositories-in-rhel-8_satellite[Configuring the Base Operating System with Offline Repositories in RHEL 8] procedure in _{InstallingServerDisconnectedDocTitle}_.
```

Comment:
If you want, you could use `{RHEL}` instead of RHEL (same below). your call.

Reply:
I have no problem making the change, but shouldn't we reference the sections by their exact names?

Reply:
yeah, that makes sense. So no change necessary.